features/storage/csi.feature

```diff
@@ -37,3 +37,16 @@ Feature: CSI testing related feature
       | /mnt/cinder/hello |
     Then the step should succeed
     And the output should contain "Hello OpenShift Storage"
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-30787
+  @admin
+  Scenario: CSI images checking in stage env
+    Given I use the "csihostpath" project 
```

Comment:
@chao007, I believe you need to be admin to run this,  You are missing the step `Given I switch to cluster admin pseudo user
`

Reply:
@chao007 is `csihostpath` project a system project?  I don't see it defined anywhere.

Reply:
@pruan-rht This case is for stage test, to check the operator installed in flexy job post action.